src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3588,6 +3302,13 @@ public sealed partial class InsufficientMemoryException : System.OutOfMemoryExce
         public static System.IntPtr MaxValue { get { throw null; } }
         public static System.IntPtr MinValue { get { throw null; } }
         public static int Size { get { throw null; } }
+        static nint System.IAdditiveIdentity<nint,nint>.AdditiveIdentity { get { throw null; } }
```

Comment:
Noting that `GenerateReferenceSource` is not `nint` vs `IntPtr` aware and will not pick the appropriate of the type vs the keyword based on what's in source.

It happens to error in most of the right places if they don't line up due to the difference in attributes, however. It would just be nice if the task generated the right source from the start.